docs/application/dotnet/samples/appfw/overview.md

```diff
@@ -173,10 +173,6 @@ for TD:
 			<ul>
 				<li><a href="/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Messages.MessagePort.html" target="_blank">Class MessagePort</a></li>
 			</ul></p>
-			<p>In addition, there is similar native sample application.<br>
-			<ul>
-				<li><a href="/development/sample/native/AppFW/%28Tutorial%29_Message_Port" target="_blank">Native version</a></li>
-			</ul></p>
 			</td>
 		</tr>
 		<tr>
```

Comment:
Line 187 can we use the following as an image?
![NoUI](https://user-images.githubusercontent.com/43839737/120691779-86416980-c4c4-11eb-8e4a-7b56fb0130e3.png)

Reply:
I've changed No UI only to Image file.

Reply:
@sgchoi5 I could not get you.
![image](https://user-images.githubusercontent.com/43839737/120845244-2911ea80-c58e-11eb-985d-60ce248b4e8c.png)
I was just asking about this.

Reply:
@nawab-samsung How about this ?
Image file with black background color with text like

No UI Available => Service application - No UI Available    or
                               Service Application - No UI Available

Reply:
That's fine, great thanks for applying this.

Reply:
@sgchoi5 sorry but the spelling is incorrect in round one 

Serivce--> Service 

Please, can you let me know if the position of the circle UI image is fine. 
![SharedScreenshot](https://user-images.githubusercontent.com/43839737/120961953-6dfb7400-c77c-11eb-837e-7344af1999c6.jpg)